### DISCUSSION TITLE
About the approach of “dominant” and “mosaic”

### DISCUSSION THREAD

**Discussion by pjt12138** [NCAR/hrldas#157] [Q&A] [answered]
Hi,  Does HRLDAS incorporate the "mosaic" parameterization scheme as described by Dan Li in 2013? Where can I locate the code file for this scheme? Additionally, where can I find the code implementing the "dominant" approach for identifying surface parameters? Thank you very much for your answer.

Li, D., E. Bou-Zeid, M. Barlage, F. Chen, and J. A. Smith (2013), Development and evaluation of a mosaic approach in the WRF-Noah framework, J. Geophys. Res. Atmos., 118, 11,918–11,935, doi:[10.1002/2013JD020657].

**Comment by cenlinhe** [comment, marked as answer]
HRLDAS or systems using Noah-MP in general does not include "mosaic" scheme. It uses "dominant" approach. The surface parameters are in the Noah-MP parameter table (NoahmpTable.TBL) here: https://github.com/NCAR/noahmp/tree/master/parameters
We have plan to include "mosaic" scheme to Noah-MP in the future but not this year.
